addon/data/postmessageProxyFrame.js

```diff
@@ -0,0 +1,17 @@
+const TELEMETRY_QUEUE_POLL_INTERVAL = 5000;
+
+// Periodically poll for queued telemetry pings
+setInterval(function() {
+  document.getElementById('tp-proxy').contentWindow.postMessage({
+    op: 'fetchTelemetryPings'
+  }, '*');
+}, TELEMETRY_QUEUE_POLL_INTERVAL);
+
+// Listen for postMessage
+window.addEventListener('message', function(ev) {
```

Comment:
Ah, so this addon event listener will be getting the messages from testpilot.firefox.com ? So maybe that's the one we want to enfroce?